### PR TITLE
✨ updates file repo filed display names

### PR DESCRIPTION
**Pull request by hlminh2000**
**Description of changes**

updates file repo filed display names

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [ ] Manual testing of UI feature.
- [ ] Add copyrights for new files
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: src/resources/arranger_es_metadata.json [hunk 1]
```diff
@@ -830,7 +830,7 @@
             "displayValues": {},
             "quickSearchEnabled": false,
             "field": "analysis.analysis_id",
-            "displayName": "Analysis Analysis Id",
+            "displayName": "Analysis Id",
```
Comment (rosibaj) [inline src/resources/arranger_es_metadata.json:833]: ID not Id (for all ids)

File: src/resources/arranger_es_metadata.json [hunk 2]
```diff
@@ -914,7 +914,7 @@
             "displayValues": {},
             "quickSearchEnabled": false,
             "field": "analysis.workflow.workflow_name",
-            "displayName": "Analysis Workflow Workflow Name",
+            "displayName": "Analysis Workflow Name",
```
Comment (rosibaj) [inline src/resources/arranger_es_metadata.json:917]: Workflow Name

File: src/resources/arranger_es_metadata.json [hunk 3]
```diff
@@ -926,7 +926,7 @@
             "displayValues": {},
             "quickSearchEnabled": false,
             "field": "analysis.workflow.workflow_version",
-            "displayName": "Analysis Workflow Workflow Version",
+            "displayName": "Analysis Workflow Version",
```
Comment (rosibaj) [inline src/resources/arranger_es_metadata.json:929]: Workflow Version

File: src/resources/arranger_es_metadata.json [hunk 4]
```diff
@@ -1166,7 +1166,7 @@
             "displayValues": {},
             "quickSearchEnabled": false,
             "field": "file.index_file.md5sum",
-            "displayName": "File Index File Md 5 Sum",
+            "displayName": "Index File Md 5 Sum",
```
Comment (rosibaj) [inline src/resources/arranger_es_metadata.json:1169]: Md5sum no spaces

File: src/resources/arranger_es_metadata.json [hunk 5]
```diff
@@ -1190,7 +1190,7 @@
             "displayValues": {},
             "quickSearchEnabled": false,
             "field": "file.index_file.object_id",
-            "displayName": "File Index File Object Id",
+            "displayName": "Object ID",
```
Comment (rosibaj) [inline src/resources/arranger_es_metadata.json:1193]: sorry missed this before

consistency with above fields:
Index File Object ID

Comment (hlminh2000) [reply src/resources/arranger_es_metadata.json]: did `Index File Size` too :)

**Pull request merged**